Library/DiscUtils.Dmg/UdifBuffer.cs

```diff
@@ -300,7 +300,7 @@ private void LoadRun(CompressedRun run)
                             new BZip2DecoderStream(new SubStream(_stream, run.CompOffset, run.CompLength),
                                 Ownership.None))
                     {
-                        StreamUtilities.ReadFully(ds, _decompBuffer, 0, toCopy);
+                        StreamUtilities.ReadExact(ds, _decompBuffer, 0, toCopy);
```

Comment:
Reads a number of bytes per run specification. Did not throw before, does now, and should.